src/traces/scattergl/attributes.js

```diff
@@ -12,7 +12,6 @@ var scatterAttrs = require('../scatter/attributes');
 var colorAttributes = require('../../components/colorscale/color_attributes');
 
 var DASHES = require('../../constants/gl2d_dashes');
-var MARKERS = require('../../constants/gl2d_markers');
```

Comment:
Can you remove this file completely while at it?

Reply:
ok